docs: Update README, website & package description (#7)
bloomberg/ts-blank-space: website/index.md

throw Error();
```

> That's because TypeScript is performing "down-leveling" to convert your modern JavaScript into something that can run on older JavaScript engines. This moves the code around without also generating sourcemaps, which confuses error stacks and breakpoint locations.

Reviewer: TypeScript _does_ produce sourcemaps. The new sentence implies this is not the case.

```suggestion
That's because TypeScript is performing "down-leveling" to convert your modern JavaScript into something that can run on older JavaScript engines. Because this moves the code around error stacks and breakpoint locations would be incorrect; so sourcemaps are also generated.
```
